sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1036,7 +1036,6 @@ def _add_argparse_args(cls, parser):
             'the command line.'))
     parser.add_argument(
         '--prebuild_sdk_container_engine',
-        choices=['local_docker', 'cloud_build'],
```

Comment:
Could you update the document to mention the built-in supported types and the example usage of extension?

Reply:
Does that cover what you were thinking of? I think if people are aware of the option and go to the trouble of implementing a subclass they will realise what build key to provide.